src/views/explore/explore.jsx

```diff
@@ -27,24 +30,29 @@ var Explore = injectIntl(React.createClass({
             stories: 'stories'
         };
         var typeOptions = ['projects','studios'];
+        var modeOptions = ['trending', 'popular', 'recent', ''];
 
         var pathname = window.location.pathname.toLowerCase();
         if (pathname[pathname.length - 1] === '/') {
             pathname = pathname.substring(0, pathname.length - 1);
         }
-        var slash = pathname.lastIndexOf('/');
-        var currentCategory = pathname.substring(slash + 1,pathname.length);
-        var typeStart = pathname.indexOf('explore/');
-        var type = pathname.substring(typeStart + 8,slash);
-        if (Object.keys(categoryOptions).indexOf(currentCategory) === -1 || typeOptions.indexOf(type) === -1) {
+        var options = pathname.split('/');
+        var type = options[2];
+        var currentCategory = options[3];
+        var currentMode = options.length > 4 ? options[4] : '';
```

Comment:
I think it'd make sense to have the default `'trending'` mode be applied here rather than below in the api request, i.e. here it would say:
`var currentMode = options.length > 4 ? options[4] : 'trending';`

Also – that might make it a bit more clear in the dropdown what the default sort is.